web/src/compose_banner.ts

```diff
@@ -137,12 +137,6 @@ export function show_stream_does_not_exist_error(stream_name: string): void {
     append_compose_banner_to_banner_list(new_row, $("#compose_banners"));
     hide_compose_spinner();
 
-    // A copy of `compose_recipient.open_compose_stream_dropup()` that
-    // can't be imported due to typescript and import circles.
-    // TODO: Once we use stream IDs, not names, as the fundamental
-    // compose_state storage for streams, this error will be impossible.
-    if ($("#id_compose_select_recipient").hasClass("open")) {
-        return;
-    }
-    $("#id_compose_select_recipient button").trigger("click");
+    // Open stream select dropdown.
+    $("#compose_select_recipient_widget").trigger("click");
 }
```

Comment:
Is it still possible for this error to happen? We can deal with that as a follow-up, but now that it's not a plain-text input, I'm not sure how you get a stream that doesn't exist in there. (I guess maybe the stream get's deleted while it's open?)

Reply:
> I guess maybe the stream get's deleted while it's open?

or just server and client not being in sync? It might be just better to have it than not.